Cargo.toml

```diff
@@ -54,6 +54,7 @@ fuel-abi-types = "0.7.0"
 futures = "0.3.29"
 hex = { version = "0.4.3", default-features = false }
 itertools = "0.12.0"
+path-clean = "1.0.1"
```

Comment:
I audited the source of `path-clean`: it's a safe, `std`-augmenting function with a set of tests and no extra dependencies.
P.S. at some point we should have a centralized repo of our crate audits, with something like `cargo-audit`